contrib/ci/qt5-thread-test/meson.build

```diff
@@ -0,0 +1,23 @@
+project('qt5-thread-test', 'cpp',
+  license : 'LGPL-2.1+',
+)
+add_project_arguments('-fPIC', language : 'cpp')
+qt5core = dependency('Qt5Core')
+qt5concurrent = dependency('Qt5Concurrent')
+glib2 = dependency('glib-2.0')
+gio2 = dependency('gio-2.0')
+fwupd = dependency('fwupd')
```

Comment:
just to clarify; this is the installed fwupd version rather than the "just built" version, right?

Reply:
Yeah it's explicitly run after install step.